livekit-agents/livekit/agents/stt/stt.py

```diff
@@ -76,14 +82,17 @@ class SpeechStream(ABC):
     class _FlushSentinel:
         pass
 
-    def __init__(self):
+    def __init__(self, *, timeout: float | None = 10.0):
         self._input_ch = aio.Chan[Union[rtc.AudioFrame, SpeechStream._FlushSentinel]]()
         self._event_ch = aio.Chan[SpeechEvent]()
+        self._req_ch = aio.Chan[None]()
```

Comment:
req_ch is confusing. Is there a way to keep this logic only inside stt/tts base class?

for e.g inside push_frame & flush?

Reply:
Or if this isn't possible, we could still add a private function that serves as a callback for the baseclass logic like:

self._request_started()